fbrp/src/fbrp/tool/__init__.py

```diff
@@ -0,0 +1,31 @@
+import click
```

Comment:
Did you end up using click in the final implementation?

Reply:
oh, I get it,,,

Reply:
Thanks for catching the unused import!
Removed.

I switched from `argparse` because it was much cleaner and easier to use. Especially for subcommands.
`click` provides auto-complete out of the box.